src/index.js

```diff
@@ -0,0 +1,96 @@
+/* eslint-disable consistent-return */
+/* eslint-disable no-use-before-define */
+import {
+  showCity, showTempearture, showIcon,
+  showDescription, showHumidity, showWind, searchCity,
+  searchCityBtn, showCountry, weatherContainer,
+  showBodyBckImg, showTempDegButton,
+} from './dom';
+
+const API_KEY = '70c8519b5e37ccac35bcbea2d8b91b5b';
+let temperatureDegButton = showTempDegButton;
+
+async function fetchWeather(city) {
```

Comment:
Kindly use ES6 arrow syntax instead of the `function` keyword we have currently :+1: